openvpn/hideme/ca.pem

```diff
@@ -0,0 +1,57 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This file doesn't seem necessary. There is no reference to it from the .ovpn config file, try without?